internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -217,6 +218,13 @@ func (rp *SQLiteRegistryPod) podForBundleRegistry() (*corev1.Pod, error) {
 			Namespace: rp.cfg.Namespace,
 		},
 		Spec: corev1.PodSpec{
+			SecurityContext: &corev1.PodSecurityContext{
+				RunAsNonRoot: pointer.Bool(false),
+				RunAsUser:    pointer.Int64(0),
```

Comment:
Why do we need the RunAsUser:    pointer.Int64(0) here?

Reply:
This is a deprecated codepath that handles SQLite based index images, so left this intact.

Reply:
So far, the SQLite images were run using the root user always, so this change just ensures it runs the same as it always has. By default if nothing is specified, pod will run as `root`. Does that clarify your question @camilamacedo86?

Reply:
Merging this PR for now, will address any potential concerns in a follow-up.